### PR TITLE
Set support expectations more clearly

### PR DESCRIPTION
**Pull request by bethaitman**
This PR:

- Removes mention of email support
- Adds expectations about

  - timing
  - public support only
  - level of questions we're happy to answer
- Adds suggestion that people can pay for support if they want more

I'm also considering adding some "help us to help you" stuff about asking good questions, what information we need etc, but I think that this PR can be standalone.

### CODE REVIEW

File: docs/source/support/support.rst [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
+
+  - Join the DAML community on `Slack <https://damldriven.slack.com/sso/saml/start>`_.
+  - Ask us for help using the `Support <javascript:open_feedback()>`_ link on any documentation page.
+
+When you're in the community Slack or on Stack Overflow, please keep to the `Code of Conduct <https://github.com/digital-asset/daml/blob/master/CODE_OF_CONDUCT.md>`__. 
+
+Support expectations
+--------------------
+
+For community users (ie on Slack and Stack Overflow):
```
Comment (stefanobaghino-da) [inline docs/source/support/support.rst]:
```suggestion
For community users (i.e. on Slack and StackOverflow):
```

File: docs/source/support/support.rst [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
+
+- **Timing**: We'll try to get back to you in two business days.
```
Comment (stefanobaghino-da) [inline docs/source/support/support.rst]:
```suggestion
- **Timing**: You can enjoy the support of the community, which is provided for you out of their own good will and free time. On top of that, a Digital Asset employee will try to reply to unanswered questions within two business days.
```

File: docs/source/support/support.rst [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
+
+  Our main offices are in Zurich and New York, and we don't consider public holidays in those locations to be business days.
```
Comment (stefanobaghino-da) [inline docs/source/support/support.rst]:
```suggestion
  Business days are affected by public holidays. Engineers contributing to DAML are mostly located in Zurich and New York, so please be mindful of the public holidays in those locations (`timeanddate.com <https://www.timeanddate.com>`_ maintains an unofficial list of holidays for both `Switzerland <https://www.timeanddate.com/holidays/switzerland/>`_ and the `United States <https://www.timeanddate.com/holidays/us/>`_).
```

File: docs/source/support/support.rst [hunk 4]
```diff
@@ -1,21 +1,36 @@
...
+- **Public support**: We only offer public support - for example, on the ``#public`` channel in `Slack <https://damldriven.slack.com/sso/saml/start>`_. 
 
-	If you're not sure what makes a good question, take a look at `this checklist <https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/>`_. 
-Help and feedback: Slack, or Support widget
-	If you want to give feedback, or have questions that aren't right for Stack Overflow, you can: 
+  We can't answer questions in private messages or over email, so please only ask questions in the ``#public`` channel.
+- **Level of support**: We're happy to answer questions about error messages you're encountering, or discuss DAML design questions. However, we can't provide more extensive consultation on how to build your DAML application.
```
Comment (stefanobaghino-da) [inline docs/source/support/support.rst]:
```suggestion
- **Level of support**: We're happy to answer questions about error messages you're encountering, or discuss DAML design questions. However, we can't provide more extensive consultation on how to build your DAML application or the languages, frameworks, libraries and tools you may use to build it.
```

File: docs/source/support/support.rst (hunk 2 above)
Comment (neil-da) [reply docs/source/support/support.rst]: I like this phrasing a lot.

File: docs/source/support/support.rst (hunk 3 above)
Comment (neil-da) [reply docs/source/support/support.rst]: 👍

File: docs/source/support/support.rst (hunk 1 above)
Comment (bethaitman) [reply docs/source/support/support.rst]: Actually, it is Stack Overflow! And following my usual style guide not to put the dots in ie and eg

**Comment by bethaitman** [comment]
Great suggestions, thank you @stefanobaghino-da

**Pull request merged**